config/deploy/olm/kustomization.yaml

```
@@ -0,0 +1,8 @@
+# Adds namespace to all resources.
+namespace: authorino-system
```

For consistency with the other supported installation methods:

```suggestion
namespace: authorino-operator
```